test/adapters/kagome/cmake/HunterConfig.cmake

```diff
@@ -17,6 +17,11 @@
 
 # TODO: Move away from custom kagome branch for testing
 hunter_config(kagome
-    URL https://github.com/soramitsu/kagome/archive/af5c718a05a3b734291dd2c4f574c8dffb76ad46.tar.gz
-    SHA1 81f456bb92d06e7b5861dacb66d4ff77515dadf6
+    URL https://github.com/soramitsu/kagome/archive/219d46ff77ae8b77974373653ce25e720ef0645f.tar.gz
+    SHA1 1a0827b729b4dc1e40e37887df53f05bbcac3607
```

Comment:
Do we still rely on a custom branch or is the version in `soramitsu-hunter` just to outdated?

Reply:
Ok, just took a better look. It seems to be a recent branch of master with single commit that adds a version to the CMake file. Will that be merged into master?